_episodes/13-looking-up-data.md

```diff
@@ -148,10 +148,12 @@ For more information on using Reconciliation services see [https://github.com/Op
 >* This will create a new column that contains the VIAF ID for the matched entity
 {: .challenge}
 
-## Extensions
+## Callout Extensions
```

Comment:
As per #189 could you move this information about Extensions to be after the mention of the VIB-BITs extension below?

Reply:
To get this to display correctly it needs to be something like:

```
>## Extensions
>The functionality in OpenRefine can be enhanced by ‘extensions’ which can be downloaded and installed to add functionality to your OpenRefine installation.
>A list of Extensions (not necessarily complete) is given on the OpenRefine downloads page at [http://openrefine.org/download.html](http://openrefine.org/download.html).
{: .callout}
```

The '>' at the start of each line is needed as it is how the lines are grouped together to appear in the "callout".

You can see an existing example of a callout in this file https://raw.githubusercontent.com/LibraryCarpentry/lc-open-refine/gh-pages/_episodes/02-importing-data.md ("what kinds of data can I import") and you can see the result in https://librarycarpentry.org/lc-open-refine/02-importing-data/index.html

Reply:
I think I made these changes.  I'm not able to see that tab anymore.  I'm digging around.